pkg/serverless/tags/tags.go

```diff
@@ -227,10 +227,14 @@ func getRuntimeFromOsReleaseFile(osReleasePath string) string {
 		log.Debug("could not read os-release file")
 		return ""
 	}
-	regExp := regexp.MustCompile(`PRETTY_NAME="Amazon Linux 2"`)
+	regExp := regexp.MustCompile(`NAME="Amazon Linux"`)
```

Comment:
These `MustCompiles` all really must be globals. That way they're only compiled once.